
{'date': 'Wed Mar 13 2019 18:37:13.414 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 5
  def
      ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 25 2019 16:07:10.804 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 8
    Historia()
  module <module> line 6
    	cenaQarto = Cena()
TypeError: 'module' object is not callable
'''},
{'date': 'Mon Mar 25 2019 18:41:07.236 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 8
    Historia()
  module <module> line 6
    	cenaQuarto = Cena(img = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTCx-MkZzcdk8uwwRpah7bUzH65QoU0dIxen-7bJp9COzSLi1CR")
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Apr 03 2019 18:46:47.393 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
  module <module> line 7
    	snoopy = Elemento (img = Elemento_cachorro,
NameError: name 'Elemento_cachorro' is not defined
'''},
{'date': 'Wed Apr 03 2019 18:47:33.747 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
  module <module> line 7
    	snoopy = Elemento (img = Elemento_snoopy,
NameError: name 'Elemento_snoopy' is not defined
'''},
{'date': 'Wed Apr 03 2019 18:48:15.549 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
  module <module> line 7
    	snoopy = Elemento (img = Elemento_snoopy,
NameError: name 'Elemento_snoopy' is not defined
'''},
{'date': 'Wed Apr 17 2019 18:28:32.692 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  txtsnoopy = Texto( cenaQuarto,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Apr 17 2019 18:28:59.82 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  txtsnoopy = Texto( cenaQuarto,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Apr 17 2019 18:30:07.781 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    Historia()
  module <module> line 13
    	snoop.vai = txtSnoopy.vai
NameError: name 'snoop' is not defined
'''},
{'date': 'Mon Apr 29 2019 18:45:51.457 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  "As school kids went for an excursion. When they reached the cave, they lit up like candles and began to explore the mountain. Tom and Amy got lost, but found Injun Joe hiding the treasure.
                                                                                                                                                                                               ^
SyntaxError: EOL while scanning string literal
'''},